hudi-client/hudi-flink-client/src/main/java/org/apache/hudi/metadata/FlinkHoodieBackedTableMetadataWriter.java

```diff
@@ -133,7 +133,7 @@ protected void commitInternal(String instantTime, Map<String, HoodieData<HoodieR
       metadataMetaClient = HoodieTableMetaClient.reload(metadataMetaClient);
     }
 
-    compactIfNecessary(writeClient);
+    compactIfNecessary(writeClient, "");
```

Comment:
can we make 2nd arg as Option<String> 
for table version 6, we can validate that this arg is present

Reply:
Addressed